libminifi/src/core/flow/StructuredConfiguration.cpp

```diff
@@ -989,6 +989,9 @@ std::string StructuredConfiguration::getOptionalField(const Node& node, const st
     return default_value;
   }
 
+  if (result.isSequence()) {
+    return (*result.begin()).getString().value();
```

Comment:
What if it's empty? Is that not possible here?

Reply:
Good point, I think it's possible and we can use the default in that case. Added a fix and test in d86c2b9145b3b7f0c44ec05d7d2c66bc28b321a6